dirtify.py

```diff
@@ -60,41 +62,22 @@
 if '.DS_Store' in images:
     images.remove('.DS_Store') # For macOS: Skip invisible Desktop Services Store file.
 
-for i, img in tqdm(enumerate(images), total=len(images)):
-    filename = f"{raw_dir}/{images[i]}"
-    img = cv2.imread(filename, cv2.IMREAD_COLOR)
-    
-    # resize and crop to make a 64 x 64 image
-    in_h = img.shape[0]
-    in_w = img.shape[1]
-    scale = max(out_h / in_h, out_w / in_w)
-
-    # resize...
-    img = cv2.resize(img, None, fx = scale, fy = scale, interpolation = cv2.INTER_AREA)
-    mid_h = scale * in_h / 2
-    mid_w = scale * in_w / 2
-
-    # ...and crop
-    img = img[round(mid_h - out_h/2):round(mid_h + out_h/2), round(mid_w - out_w/2):round(mid_w + out_w/2)]
-    out_name = os.path.splitext(images[i])[0] + ".png" # Change the extension to png (for lossless image)
-    cv2.imwrite(f"{clean_dir}/{out_name}", img)
-
-    # Add that dirt!
-    dirty = img
-    
+# Dirtify function
```

Comment:
I can't comment on the lines with the directory names; this is just the closest line. I thought we fixed the directory names to use `./` instead of `../`?

Reply:
I made that change in this refactorDirtify branch, but I'm just using `"filename"` instead of `"./filename"`.

Reply:
Okay cool, maybe that was a separate commit or something because I think the diff was still showing `../filename` when I reviewed the PR :) I see it's fixed in the later merge today.